src/tablelist.jsx

```diff
@@ -39,13 +41,13 @@ class IssueTable extends React.Component {
 
     handleKeyPress(event) {
         switch (event.keyCode) {
-            case 99:
+            case this.KEY_CODE_c:
                 this.toggleCoreIssues();
                 break;
-            case 97:
+            case this.KEY_CODE_a:
                 this.toggleHandeledIssues();
```

Comment:
Saknas en break :-) får du this.KEY_CODE_a som unbound i loggen.

Reply:
Aj.. Bra fångat!